contents/handbook/growth/sales/crm.md

```diff
@@ -81,28 +81,52 @@ You can also just manually add a user to HubSpot under 'Contacts'. When creating
 ## How we calculate Ideal Customer Profile (ICP)
 
 ICP scoring helps us to focus our efforts on those customers who are likely to help us hit our growth targets quickly.
-We use [Clearbit](https://clearbit.com/) to enhance our contact information as they are created and then compute a score out of 5 based on the following parameters:
+We use [Clearbit](https://clearbit.com/) to enhance our contact information as they are created and then compute a score out of 25 in HubSpot based on the following parameters:
```

Comment:
And anything 13 or over we consider a high ICP customer? (I'm assuming we need to use whole numbers in order to break this down in PH)